.github/workflows/build.yml

```diff
@@ -1,8 +1,6 @@
 name: Build
 
 on:
-  push:
-    branches: [ "main" ]
   pull_request:
     branches: [ "main" ]
```

Comment:
We should rename build.yml to something like pr.yml and release.yml to main.yml.